### ISSUE TITLE
 sticky notifications come back again!

### ISSUE THREAD

**Issue by 532910** [vector-im/riot-android#2401]
Really I've alreade posted two regeshakes for stacktrace. This bug is for attention attraction and discussion.

**Comment by nja0087** [comment]
Yep, same for me. And lost the ability to get GCM pushed notifications
